my-app/src/frontend/components/navbar/Navbar.js

```diff
@@ -4,12 +4,14 @@ import { AiFillHeart } from "react-icons/ai";
 import "./navbar.css";
 import "../../../App.css";
 import { Link, useNavigate } from "react-router-dom";
-import { useAuth } from "../../contexts/auth-context";
+import { useAuth, useCart, useWishlist } from "../../contexts/index";
 
 function Navbar() {
   const { authState, authDispatch } = useAuth();
```

Comment:
you can deconstrute token here itself, you just have to pass token directly from auth-context.